main-command/src/main/scala/sbt/internal/client/NetworkClient.scala

```diff
@@ -330,6 +330,18 @@ class NetworkClient(
 
         val cmd = arguments.sbtLaunchJar match {
           case Some(lj) =>
+            if (log) {
+              val sbtScript = if (Properties.isWin) "sbt.bat" else "sbt"
+              console.appendLog(Level.Warn, s"server is started using sbt-launch jar directly")
+              console.appendLog(
+                Level.Warn,
+                "this is not the recommended way: .sbtopts and .jvmopts files are not loaded"
+              )
+              console.appendLog(
+                Level.Warn,
+                s"make sure $sbtScript is accessible from $$PATH and run 'sbt bspConfig'"
```

Comment:
I thought this was not specific to BSP and was about to suggest a more generic message, but I guess it is, effectively. I tried to run `sbtn` without `sbt` in the path, and in that case the process just dies without any logs, most likely because the default case below is evaluated and the script it not found.

Reply:
`--sbt-launch-jar` is only usefull for IntelliJ to be able to connect with BSP using its own bundled jar. Otherwise it would have probably never been added.

Reply:
I see, https://github.com/sbt/sbt/pull/6177/files shows indeed that it was adhoc (and not documented).

Reply:
nitpicking, but I guess upgrading the sbt script to 1.6.x or 1.5.6 (not sure if this is meant to be backported?) is the preferred recommendation?
```suggestion
                s"either upgrade $sbtScript or make sure it is accessible from $$PATH, and run 'sbt bspConfig'"
```